### PR TITLE
sc 760 - wizard seed details: add missing fields

### PR DESCRIPTION
**Pull request by hiaux0**
## What was done
Seed config
- feat(seed.config): add isPermissioned prop
- feat(seed.config): add seedTip prop

SeedService
- feat(services): update isPermissioned prop
- feat(services): update where tip is taken from

Wizard Seed Details
- feat(wizard.seedDetails): add permissionedToggle
- feat(wizard.seedDetails): add tipping

Wizard Summary
- feat(wizard.summary): add permissoned toggle
- feat(wizard.summary): take tipping from object
- styles(wizard.summary): cursor:default to toggle

### Before
missing sections

### After
added sections, and they should be connected (stage 4 with summary page)

### CODE REVIEW

File: src/newLaunch/permissoned.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template bindable="checked, toggle, disabled">
+  <div class="infoInput">
+    <div class="labeledQuestion">
+      <label for="geoBlock">Set Permission-less SEED</label>
```
Comment (giladt) [inline src/newLaunch/permissoned.html:4]: Please rename the `geoBlock` to something more descriptive. Maybe `permissionedToggle`?
Though, the best could be a reusable component for both...

File: src/services/SeedService.ts [hunk 2]
```diff
@@ -241,8 +241,8 @@ export class SeedService {
       // convert from ISO string to Unix epoch seconds
       Date.parse(config.launchDetails.endDate) / 1000,
       [config.launchDetails.vestingPeriod, config.launchDetails.vestingCliff],
-      !!config.launchDetails.whitelist,
-      toWei(SeedService.seedFee),
+      config.launchDetails.isPermissoned,
+      toWei(config.launchDetails.seedTip ?? 0.0),
```
Comment (giladt) [inline src/services/SeedService.ts:245]: How is `seedTip` stored in the contract? Is it BigNumber (uint256)? couldn't find any reference for that.
Should it be on a scale of 0 - 100 or 0.0 - 1.0?

**Review by giladt — CHANGES_REQUESTED** [review]

File: src/newLaunch/seed/stages/stage4.html [hunk 3]
```diff
@@ -273,6 +279,27 @@
...
+            <div class="seedTipping">
+              <numeric-input
+                placeholder="1"
+                id="vestingPeriod"
+                not-wei
+                value.bind="launchConfig.launchDetails.seedTip">
+              </numeric-input>
+              <div>%</div>
+            </div>
+          </div>
```
[the remark this answers is not in the document]
Comment (hiaux0) [reply src/newLaunch/seed/stages/stage4.html:301]: good catch, for now is optional

File: src/newLaunch/seed/stages/stage4.ts [hunk 4]
```diff
@@ -136,7 +140,7 @@ export class Stage4 extends BaseStage<ISeedConfig> {
     // Save the admin address to wizard state in order to persist it after launchConfig state is cleared in stage7
     this.wizardState.launchAdminAddress = this.launchConfig.launchDetails.adminAddress;
     // TODO: Refactor after BE enables allowlists:
-    // this.wizardState.whiteList = this.launchConfig.launchDetails.whitelist; 
+    // this.wizardState.whiteList = this.launchConfig.launchDetails.whitelist;
     this.wizardState.launchStartDate = this.launchConfig.launchDetails.startDate;
   }
```
[the remark this answers is not in the document]
Comment (hiaux0) [reply src/newLaunch/seed/stages/stage4.ts:146]: will create follow up ticket

File: src/services/SeedService.ts (hunk 2 above)
Comment (hiaux0) [reply src/services/SeedService.ts:245]: Let's go with what the tipping input spits out to us (eg. what `value` from the input returns)

**Pull request merged**